common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,53 +1,60 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset role="${input_type}group" aria-label="${label}">
-        % for choice_id, choice_description in choices:
-        <label for="input_${id}_${choice_id}"
-            ## If the student has selected this choice...
-            % if input_type == 'radio' and ( (isinstance(value, basestring) and (choice_id == value))  or  (not isinstance(value, basestring) and choice_id in value) ):
+    <fieldset>
+      % if response_data and response_data['label']:
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+      % else:
+        <legend>Question</legend>
```

Comment:
HI @muzaffaryousaf is choicegroup=dropdown? 
https://openedx.atlassian.net/wiki/display/A11Y/Dropdown?focusedCommentId=97452470#comment-97452470
We don't yet have consensus that this template text is the way to go. @sstack22 is back in the office though so maybe I'll be able to go ahead with doing templates for the other problem types today.